org.cloudcompare.CloudCompare.yaml

```diff
@@ -63,38 +71,19 @@ cleanup:
   - /share/pcl-*
   - /share/pkgconfig
   - /share/proj
-  - /share/rdblib
 
   - /include
 
   - /doc
 
-  - /rdblib
-
-  - /cork
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
```

Comment:
Where does this file come from? Is this part of the upstream repo?

Reply:
no from the PyQT BaseApp